hw/top_earlgrey/data/ip/chip_spi_host_testplan.hjson

```diff
@@ -16,9 +16,81 @@
+            - The testbench that mimics a physical SPI device should measure the clock to verify that the frequencies for each speed mode is correct.
+            This can be done by checking timing of the beginning and the end of the packet.
             '''
+      features: [
+        "SPIHOST.RATE.STANDARD",
+        "SPIHOST.RATE.DUAL",
+        "SPIHOST.RATE.QUAD",
+      ]
       stage: V2
+      si_stage: SV2
       tests: ["chip_sw_spi_host_tx_rx"]
     }
+    {
+      name: chip_sw_spi_host_pass_through
+      desc: '''Verify that the SPI host can be configured in pass through mode.
+
+            - Connect the SPI host to a serial NOR flash device.
+            - Configure an SPI device and that SPI host in pass through mode.
+            - Use the testbench's SPI host to write a word to the flash through the passthrough SPI device and host in OpenTitan.
+            - Read that word back.
+            - Check that the word read back is the same as the one written.
+            '''
+      features: [
+        "SPIHOST.USECASE.SERIALNORFLASH",
+        "SPIHOST.USECASE.PASSTHROUGH",
+      ]
+      stage: V3
+      si_stage: SV3
+      tests: []
+    }
+    {
+      name: chip_sw_spi_host_configuration
+      desc: ''' Verifiy that the polarity and clock divider configurations work appropriately.
+
+            1. Send a packet with no clock division and no polarity switch.
+            2. Send a packet with no clock division and a polarity switch.
+            3. Send a packet with clock divided by two and a polarity switch.
+
+            The testbench should compare the three packets that are received, the second packet should be inverted from the first and the third should be send half as fast as the second.
+            '''
+      features: [
+        "SPIHOST.CONFIG.CPOL",
+        "SPIHOST.CONFIG.CLOCKDIV",
+      ]
+      stage: V3
+      si_stage: SV3
+      tests: []
+    }
+    {
+      name: chip_sw_spi_host_events
+      desc: ''' Verify that various events are correctly triggered by the SPI host.
+
+            - For transmit empty event:
+              - Fill the transmit queue on the Ibex side.
+              - Clear and enable the interrupt.
+              - Testbench should check that it receives the same data as Ibex has sent.
+              - Ibex should check whether it has received the transmit empty interrupt within a reasonable amount of time.
+            - For receive full event:
+              - Empty the receive queue.
+              - Clear and enable the interrupt.
+              - Testbench sends as much data as the receive queue can hold.
+              - Ibex checks that the receive full interrupt is raised.
+            - For watermark events, the test should do similar to the previous described events, but instead enable the transmit and receive watermark respectively.
+            Also, instead of using the full queue size, now use the watermark value for example half of the queue.
```

Comment:
Do you mean to use that value as the amount of data to send? If I've understood right, I think this only applies on the RX side as well?

Reply:
It seems like this last part was a bit confusing so I decided to update the transmit and receive events to include the watermark descriptions as well. I think that's clearer.